content/4_Gold/BFS.mdx

```diff
@@ -12,7 +12,9 @@ frequency: 2
 <!-- NOTE: general problems were RANDOMLY STARRED. please review! -->
 
 export const problems = {
-	sampleNormal: new Problem('CSES', 'Message Route', '1667', 'Easy', false, ['BFS']),
+	sampleNormal: new Problem('CSES', 'Message Route', '1667', 'Easy', false, [
+		'BFS',
```

Comment:
was this an accidental misenter?

Reply:
this just happens when the last argument in constructor is a list 🤷

Reply:
i c